package.json

```diff
@@ -97,14 +97,15 @@
     "babel-jest": "19.0.0",
     "babel-loader": "6.2.10",
     "babel-plugin-add-module-exports": "0.2.1",
+    "babel-plugin-syntax-dynamic-import": "6.18.0",
+    "babel-plugin-transform-async-generator-functions": "6.24.1",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-plugin-transform-unicode-property-regex": "2.0.1",
```

Comment:
I was originally basing the list of plugins off of babel#master, rather than the released versions